packages/fuel-gauge/src/transaction-response.test.ts

```diff
@@ -75,25 +83,33 @@ describe('TransactionSummary', () => {
   });
 
   it('should ensure waitForResult always waits for the transaction to be processed', async () => {
-    const destination = Wallet.generate({
-      provider,
+    const { cleanup, ip, port } = await launchNode({
+      args: ['--poa-interval-period', '10s'],
     });
```

Comment:
Why are we starting a new node just for this test?

Reply:
To verify that the subscription will wait for 10 seconds until the transaction changes status.